doc/tech/README.md

```diff
@@ -0,0 +1,148 @@
+
+<p align="center"><img src="./figures/tech_vtd.png" alt="virtio" title="Document Types" width="50%"/></p>
+<p align="center"><b>Figure 5:</b> Direct Assignment with Virtual Technology.</p>
+
+<a name="4.1.3"></a>
+#### 4.1.3 Device Sharing with SR-IOV & Virtual Technology.
+
+Unlike Direct assignment, this method allows a hardware device to be shared amongst many workloads where each workload has an exclusive access to one region of the device (known as VF). 
+
+For this method to be possible, the IO device need to support Single Root Input Output Virtualization (SR-IOV) which allows it to present itself as multiple devices, known as Physical Functions, PFs, and Virtual Functions, VFs as presented in **Figure 6**.
+
+Each of those Virtual Functions can then be independently assigned exclusively to a workload (with the appropriate support by VT-d or AMD-V from the infrastructure).
+
+Similar to the previous method ("Direct Assignment"), this method provides better performance than the para-virtualized one but lacks the flexibility and the portability sought and therefore should also not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
+
+<p align="center"><img src="./figures/tech_sriov.png" alt="sriov" title="Document Types" width="50%"/></p>
+<p align="center"><b>Figure 6:</b> Device Sharing with SR-IOV & Direct Assignment.</p>
+
+<a name="4.1.4"></a>
+#### 4.1.4 Para-Virtualization method (Hardware support)
+
+This method basically is a mixture between the software only para-virtualization method and the direct assignment method (including the device sharing method) where the frontEnd driver which is running on the workload is a standard off the shelf driver whereas the backEnd driver is implemented straight in hardware (by-passing the hypervisor with support from VT-d/AMD-V and SR-IOV in case of device sharing) as shown in **Figure 7**.
+
+Unlike the software only para-virtualized interfaces, this method provides better performance as it by-passes the hypervisor and unlike Direct Assignment methods, this method doesn’t require proprietary drivers to run in the workload and hence this method makes workloads portable.
+
+However, this method doesn’t provide the same level of flexibility as the software only para-virtualization method as migrating workloads from one host to another is more challenging due to the hardware presence and the state it holds for the workloads using it and therefore should also not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
+
+<p align="center"><img src="./figures/tech_virtio_hw.png" alt="vitio_hw" title="Document Types" width="50%"/></p>
+<p align="center"><b>Figure 7:</b> Para-Virtualization method (with hardware support).</p>
+
+<a name="4.3"></a>
+### 4.2 SmartNICs
```

Comment:
What is the intended content for sections 4.2 through 4.5? 
If it is to identify the technologies chosen or mapping to the virtualization types identified in 4.1.1 through 4.1.4, perhaps it may be best-served in the RM or RA documents because:
- Over time, as we evolve the hardware that is deployed, choices or implementations of these capabilities may change from one profile generation (basic gen1 to basic gen2, for example).  The updated RM and RA documents would list both basic gen1 and basic gen2, where basic gen1 might permit (for example) SR-IOV, and basic gen2 would introduce the use of a new driver and technology to replace it.  
- Documenting them here creates a more challenging path forward for introducing these changes on a profile-by-profile basis.

Reply:
@kedmison I added additional clarification that the intention is to explain those technologies. where Section 5 will set the policy. the intentions to remove any confusion about what do we mean by things like DA, PCIe Pass-though, SR-IOV, etc.